portfolio/src/main/webapp/index.html

```diff
@@ -49,53 +50,148 @@ <h4>Software Engineering Intern</h4>
     </div>
 
     <!-- Main Content Section -->
+
     <div id="home" class="menu">
-      <section>
-          <div class="info">
-            <div class="title"><h2>About Me</h2></div>
-              <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
-              </p>
+      <section id="home" class="landing">
+        <div class="content"> 
+          <h2>Hi there.<br> I'm <span> Kayla Leung </span></h2>
+          <button id="testbutton" class="button" onclick="document.location = '#aboutme'">About Me</button>
```

Comment:
This id does nothing!

So I very much recommend avoiding inline JavaScript whenever possible. In this particular case, your button is not actually a button - it does the action of a link. It would be more appropriate here to use an anchor tag with a href of #aboutme. (You can still style it as usual.)